BDArmory/Control/MissileFire.cs

```diff
@@ -2504,9 +2500,9 @@ IEnumerator GuardMissileRoutine(Vessel targetVessel, MissileBase ml)
                                     yield return new WaitForFixedUpdate();
                                 }
                             }
-                            if (targetVessel && ml && mlauncher.multiLauncher && mlauncher.multiLauncher.turret)
+                            if (mlauncher.multiLauncher && mlauncher.multiLauncher.turret)
                             {
-                                while (Time.time - turretStartTime < Mathf.Max(targetScanInterval / 2f, 2) && mlauncher && targetVessel && angle > mlauncher.missileTurret.fireFOV)
+                                while (Time.time - turretStartTime < Mathf.Max(targetScanInterval / 2f, 2) && mlauncher && targetVessel && angle > mlauncher.multiLauncher.turret.fireFOV)
```

Comment:
This `while` loop (and the one above it) should just `yield return wait;` instead of yielding a `new WaitForFixedUpdate()` to avoid GC allocations.